main_2023.py

```diff
@@ -1,27 +1,37 @@
 """
 For 2022-2023 UAS competition.
 """
+import argparse
 import multiprocessing as mp
 
 import cv2
+import yaml
 
 from utilities.workers import queue_proxy_wrapper
 from utilities.workers import worker_controller
 from utilities.workers import worker_manager
 from modules.detect_target import detect_target_worker
 from modules.video_input import video_input_worker
 
+# select config file from command line
+parser = argparse.ArgumentParser()
+parser.add_argument("--config", choices=["cuda", "cpu"], required=True, help="Select configuration file")
```

Comment:
Instead of choosing between configuration files, we can have the option to force CPU with `--cpu` .